.github/workflows/deploy-personal-dev-environment.yml

```diff
@@ -42,7 +41,7 @@ jobs:
       - name: Terraform apply
         run: |
           source uhd.sh
-          uhd terraform init
+          uhd terraform init:layer 20-app
```

Comment:
Not actually needed. But it's a bit more granular i.e. cause we're only deploying app-layer resources